app/services/foreman_ansible/override_resolver.rb

```diff
@@ -0,0 +1,20 @@
+module ForemanAnsible
+  # Service which resolves override values for hosts
+  class OverrideResolver
+    attr_reader :overrides, :ansible_variables
+
+    def initialize(host)
+      if host
```

Comment:
It is probably not an issue in a context of the host detail page, but the method relies on host not being nil.

Reply:
In case this is ever called with a `nil` host (e.g. in case we reuse it somewhere else), should we silently provide an empty hash or would it be better to throw an error?